src/core/mempool.rs

```diff
@@ -557,7 +558,7 @@ impl MemPoolDB {
                 &tip_block_hash,
                 next_timestamp,
             )?;
-
+            update_stx_mempool_size(available_txs.len() as i64); //promserver
```

Comment:
Are you trying to get the total size of the mempool here?  If so, then this doesn't do what you think it does.  The `get_txs_at()` method only returns the number of transactions that arrived while the given `tip_consensus_hash` and `tip_block_hash` were the chain tip.  I think you'll want to add a method that returns the total number of rows in the mempool, and then call this `update_stx_mempool_size()` method within the relayer (right after it does a mempool garbage-collection).

Reply:
Correct, the idea here was to track the current size of the mempool. I've removed this for now, will look into this more at a later time.